spec/features/1_user_signup_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature '1: user sign up' do
```

Comment:
I've not included any unit tests in my application. We weren't shown how to test DataMapper models (as opposed to pure Ruby classes) in the week's challenge - and wasn't sure how to approach it. Do you have any hints or tips?